README.md

```diff
@@ -1,212 +1,65 @@
+| [v0.7.0](https://github.com/tektoncd/dashboard/releases/tag/v0.7.0) | [Docs @ v0.7.0](https://github.com/tektoncd/dashboard/tree/v0.7.0/docs) |
+| [v0.6.1.6](https://github.com/tektoncd/dashboard/releases/tag/v0.6.1.6) | [Docs @ v0.6.1.6](https://github.com/tektoncd/dashboard/tree/v0.6.1.6/docs) |
+| [v0.6.1.5](https://github.com/tektoncd/dashboard/releases/tag/v0.6.1.5) | [Docs @ v0.6.1.5](https://github.com/tektoncd/dashboard/tree/v0.6.1.5/docs) |
+| [v0.6.1.4](https://github.com/tektoncd/dashboard/releases/tag/v0.6.1.4) | [Docs @ v0.6.1.4](https://github.com/tektoncd/dashboard/tree/v0.6.1.4/docs) |
+| [v0.6.1.3](https://github.com/tektoncd/dashboard/releases/tag/v0.6.1.3) | [Docs @ v0.6.1.3](https://github.com/tektoncd/dashboard/tree/v0.6.1.3/docs) |
+| [v0.6.1.2](https://github.com/tektoncd/dashboard/releases/tag/v0.6.1.2) | [Docs @ v0.6.1.2](https://github.com/tektoncd/dashboard/tree/v0.6.1.2/docs) |
+| [v0.6.1.1](https://github.com/tektoncd/dashboard/releases/tag/v0.6.1.1) | [Docs @ v0.6.1.1](https://github.com/tektoncd/dashboard/tree/v0.6.1.1/docs) |
+| [v0.6.1](https://github.com/tektoncd/dashboard/releases/tag/v0.6.1) | [Docs @ v0.6.1](https://github.com/tektoncd/dashboard/tree/v0.6.1/docs) |
+| [v0.6.0](https://github.com/tektoncd/dashboard/releases/tag/v0.6.0) | [Docs @ v0.6.0](https://github.com/tektoncd/dashboard/tree/v0.6.0/docs) |
+| [v0.5.3](https://github.com/tektoncd/dashboard/releases/tag/v0.5.3) | [Docs @ v0.5.3](https://github.com/tektoncd/dashboard/tree/v0.5.3/docs) |
+| [v0.5.2](https://github.com/tektoncd/dashboard/releases/tag/v0.5.2) | [Docs @ v0.5.2](https://github.com/tektoncd/dashboard/tree/v0.5.2/docs) |
+| [v0.5.1](https://github.com/tektoncd/dashboard/releases/tag/v0.5.1) | [Docs @ v0.5.1](https://github.com/tektoncd/dashboard/tree/v0.5.1/docs) |
+| [v0.5.0](https://github.com/tektoncd/dashboard/releases/tag/v0.5.0) | [Docs @ v0.5.0](https://github.com/tektoncd/dashboard/tree/v0.5.0/docs) |
+| [v0.4.1](https://github.com/tektoncd/dashboard/releases/tag/v0.4.1) | [Docs @ v0.4.1](https://github.com/tektoncd/dashboard/tree/v0.4.1/docs) |
+| [v0.4.0](https://github.com/tektoncd/dashboard/releases/tag/v0.4.0) | [Docs @ v0.4.0](https://github.com/tektoncd/dashboard/tree/v0.4.0/docs) |
+| [v0.3.0](https://github.com/tektoncd/dashboard/releases/tag/v0.3.0) | [Docs @ v0.3.0](https://github.com/tektoncd/dashboard/tree/v0.3.0/docs) |
+| [v0.2.1](https://github.com/tektoncd/dashboard/releases/tag/v0.2.1) | [Docs @ v0.2.1](https://github.com/tektoncd/dashboard/tree/v0.2.1/docs) |
+| [v0.2.0](https://github.com/tektoncd/dashboard/releases/tag/v0.2.0) | [Docs @ v0.2.0](https://github.com/tektoncd/dashboard/tree/v0.2.0/docs) |
+| [v0.1.1](https://github.com/tektoncd/dashboard/releases/tag/v0.1.1) | [Docs @ v0.1.1](https://github.com/tektoncd/dashboard/tree/v0.1.1/docs) |
+| [v0.1.0](https://github.com/tektoncd/dashboard/releases/tag/v0.1.0) | [Docs @ v0.1.0](https://github.com/tektoncd/dashboard/tree/v0.1.0/docs) |
```

Comment:
0.6.1 was the first release with docs content in the `docs` folder (not counting the extensions content in `docs/samples` which was never really promoted given it alpha/pre-alpha state). I'd be inclined to cut down this list a bit since we're advising users to pick up 0.6.1.4 or newer and don't really need to show every patch release for older versions. We could also use this to present our compatibility chart, maybe something like:


### Which version should I use?

It is **strongly recommended** to use the **v0.6.1.4** release or newer for Tekton Pipelines v0.11.x and v0.12.x and Tekton Triggers v0.4.
- This is a **critically important security release**
- Earlier versions are deprecated and should be used for **development or isolated usage only**

| Dashboard | Docs | Pipelines | Triggers |
|---|---|---|---|
| [HEAD](#) | [Docs @ HEAD](#) | v0.11.x -<br />v0.13.x | v0.5.x |
| [v0.7.0](#) | [Docs @ v0.7.0](#) | v0.11.x<br />v0.12.x | v0.5.x |
| [v0.6.1.5](#) | [Docs @ v0.6.1.5](#) | v0.11.x<br />v0.12.x | v0.4.x |
| [v0.6.1.4](#) | [Docs @ v0.6.1.4](#) | v0.11.x | v0.4.x |
| [v0.6.0](#) |  | v0.11.x | v0.3.x |
| [v0.5.3](#) |  | v0.10.x | v0.3.x |
| [v0.5.2](#) |  | v0.10.x | v0.2.x |
| [v0.5.0](#) |  | v0.10.x | v0.1 |
| [v0.4.1](#) |  | v0.8.0 | v0.1 |
| [v0.2.1](#) |  | v0.7.0 |  |
| [v0.1.1](#) |  | v0.5.2 |  |

All versions from v0.5.0 support a read-only mode.

Reply:
no v0.6.1.6 ?

Reply:
not sure how I missed 0.3.0
support for pipelines 0.8 - 0.9 and triggers 0.1

yes we should include 0.6.1.6 and 0.7.0 once they're ready (still marked as pre-release)

Reply:
ok, i guess v0.4.1 is for pipelines v0.9.0 then ?

Reply:
updated comment above, we skipped pipelines 0.9 support: https://github.com/tektoncd/dashboard/issues/1154#issuecomment-600115090

Reply:
Thanks, fixed.